MoltenVK/MoltenVK/Commands/MVKCmdRendering.h

```diff
@@ -607,6 +607,25 @@ class MVKCmdSetPolygonMode : public MVKCommand {
 };
 
 
+#pragma mark -
+#pragma mark MVKCmdSetLineWidth
+
+/** Vulkan command to dynamically set the line width. */
+class MVKCmdSetLineWidth : public MVKCommand {
+
+public:
+	VkResult setContent(MVKCommandBuffer* cmdBuff,
+						float lineWidth);
+
+	void encode(MVKCommandEncoder* cmdEncoder) override;
+
+protected:
+	MVKCommandTypePool<MVKCommand>* getTypePool(MVKCommandPool* cmdPool) override;
+
+	float _lineWidth;
+};
+
+
```

Comment:
I wonder if all the line width stuff should be hidden behind the compile-time switch.

Reply:
I thought of that, but figured it would add clutter with more conditional compiling macros. Plus, this way makes it trivial to adjust if Apple makes this call public in some future release. And if the app ignores `wideLines`, it will just result in a no-op as far as functionality is concerned.

Is your concern just the cost of carrying compiled code that cannot be used?

Reply:
I wonder how the private API usage is determined when uploading to the App Store. I feel like if they see that at some point we invoke private API, no matter if its used at runtime or not, it gets rejected? Something like objdump but for linked symbols would show setLineWidth for the MoltenVK dylib, which could cause issues I think.

Please correct me if this is not the case.

Reply:
> Is your concern just the bloat cost of carrying compiled binary code that will never be used?

Yes.